lib/support.js

```diff
@@ -41,7 +39,7 @@ function isObject(obj, type) {
  * objects - An array of objects we want to test the type of
  * type - The expected type
  */
-function assertObject(objects, type) {
```

Comment:
Touche.  I was coming from Ruby where there are "validators".  I'll revert the changes because I like this convention better.